roles/generate_manifests/templates/agent-config.yaml.j2

```diff
@@ -5,16 +5,24 @@ rendezvousIP: {{ hostvars[agent_based_installer_bootstrap_node][host_ip_keyword]
 additionalNTPSources:
     - {{ hostvars[agent_based_installer_bootstrap_node]['ntp_server'] }}
 hosts:
-{% for hostname, network_config in static_network_config.items() %}
+{% for hostname in groups['masters'] + (groups['workers'] | default([])) | sort %}
```

Comment:
By attempting to create an inventory file for this I realized that the way the code is right now, it requires a static_network_config in order to use root_device_hints/ installation_disk_path, could it be changed to loop over the hosts to either one of them?

Reply:
And I just realized that it is not true, line 18 does it without static_network_confitg at all, so all good :laughing: